frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -12,13 +13,80 @@ import { EmailAttachmentForm } from '../email-attachment';
+        {' '}<button className={classnames("button","is-text","is-paddingless","is-uppercase", isExpanded && "is-hidden")} 
+              onClick={() => toggleExpansion(true)}>
+          Learn More
+        </button>
+      </div>
+      <div className={classnames("content", !isExpanded && "is-hidden")}>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/chat.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">Before the repair-worker arrives</h3>
+            <p>Talk to anyone that you live with and let them know that a repair-worker is coming to perform the repairs that you requested.</p>
+          </div>
+        </div>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/hands.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">While the repair-worker is inside your home</h3>
+            <p>Have the repair-worker wash their hands with soap for at least 20 seconds as soon as they come into your house.</p>
+            <p>If possible, stay in a different room from where the work is being done. If a separate room is not available, maintain at least a six-foot distance from the repair-worker until the repair is completed.</p>
+          </div>
+        </div>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/wipe.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">After the repair-worker leaves</h3>
+            <p>Immediately sanitize all surfaces in your home, especially doorknobs, the sink where the repair-worker washed their hands, and any surfaces you know they have likely been in contact with.</p>
+            <p className="is-size-7">For guidance on how to thoroughly sanitize your home and a list of recommended effective cleaning products visit 
+              {' '}<OutboundLink href="https://www.cdc.gov/coronavirus/2019-ncov/prepare/cleaning-disinfection.html" target="_blank">Center for Disease Control (CDC) Guide on How to Clean and Disinfect</OutboundLink>
+            </p>
+          </div>
+        </div>
+        <div className="hero is-small is-warning">
+          <div className="hero-body has-text-centered">
+            <button className={classnames("button","is-text","is-paddingless","is-uppercase")} 
+              onClick={() => toggleExpansion(false)}>
+                Close
+            </button>
```

Comment:
I think this might actually require a bit of ARIA markup to make accessible... see for instance the `AriaExpandableButton` class in `frontend/lib/aria.tsx`.  Basically the ARIA metadata will let screen-reader users know that clicking the button will expand/collapse some area on the page, and it will also let the user know whether that content is currently expanded or collapsed, I think.

Reply:
Er, also, we might want to wrap this in a `<ProgressiveEnhancement>` component similar to the close box from #1047, since this won't really do anything for folks in compatibility mode?

Reply:
Oh also, we will want to have this be auto-expanded in compatibility mode, since the "learn more" won't actually do anything (we should definitely wrap it in a `<ProgressiveEnhancement>`) and since users won't be able to expand it, we need to just show it all expanded by default.

Reply:
Should be resolved now with a specific layout just for folks in Safe Mode. Also added ARIA markup :)